### DISCUSSION TITLE
Option to show only shortcuts when opening a new tab

### DISCUSSION THREAD

**Discussion by FernandoLins8** [dailydotdev/daily#795] [Feature Requests]
Hi there, first of all, thanks for the great app.

My issue is the following: as a user who frequently opens new tabs for work and study, the current default view of news on the Daily Dev homepage can be very distracting and is the main reason that I don't use the extension. Furthermore I rarely access it because of not having the extension on my computer and having google news accessible with only one sweep at my phone (and that's a shame because I'd really prefer dailydev if more accessibly without distracting me).

I think an option to hide the news by default and display only the user shortcuts and a button to go to news (also making possible to add more shortcuts) would greatly improve the app experience while still allowing easy access to the news with a single click.

**Comment by FernandoLins8** [comment]
Giving more details if I wasn't clear enough, my suggestion is something like the following:

* It could be added a "Non distracting mode" checkbox in the customize settings;
* When not checked the new tab dailydev home behaves as default;
* If marked, then:
  *  When a new tab is open it shows only the user shortcuts centralized occupying the same place as the first articles (this also could allow the user to have more shortcuts added);
  * Above the centralized shortcuts there are two buttons: "News" and "Shortcuts". The shortcut mode is selected by default and its the behaviour described above, while the "News" displays the default articles view.

**Reply by idoshamun** [reply]
@FernandoLins8 did you have the chance to check out our Pause New Tab feature? https://docs.daily.dev/docs/key-features/pause-new-tab

**Reply by FernandoLins8** [reply]
Yes, I've already used but is not exactly what I was looking for. Instead of setting a timer for disabling I think it would be great to just keep daily.dev always there and peek whenever possible. Anyway, thanks for replying.

**Reply by idoshamun** [reply]
Unfortunately, browsers don't provide us the API to control this setting. May I suggest using the webapp instead? https://app.daily.dev
